lib/class-wp-rest-menu-items-controller.php

```diff
@@ -0,0 +1,641 @@
+<?php
+
+/**
+ * Class WP_REST_Menu_Items_Controller
+ */
+class WP_REST_Menu_Items_Controller extends WP_REST_Posts_Controller {
+
+	/**
+	 * Get the post, if the ID is valid.
+	 *
+	 *
+	 * @param int $id Supplied ID.
+	 *
+	 * @return WP_Post|WP_Error Post object if ID is valid, WP_Error otherwise.
+	 */
+	protected function get_post( $id ) {
+		$post = parent::get_post( $id );
+		if ( is_wp_error( $post ) ) {
+			return $post;
+		}
+		$nav_item = wp_setup_nav_menu_item( $post );
+
+		return $nav_item;
+	}
+
+	/**
+	 * Creates a single post.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 *
+	 * @return WP_REST_Response|WP_Error Response object on success, or WP_Error object on failure.
```

Comment:
`@return` should be the last item in the PHPDoc: https://docs.phpdoc.org/references/phpdoc/tags/return.html

Reply:
Good spot :+1: